acceptance/openstack/containerinfra/v1/containerinfra.go

```diff
@@ -0,0 +1,94 @@
+package v1
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+	"github.com/gophercloud/gophercloud/openstack/containerinfra/v1/clustertemplates"
+	th "github.com/gophercloud/gophercloud/testhelper"
+)
+
+// CreateClusterTemplate will create a random cluster tempalte. An error will be returned if the
+// cluster-template could not be created.
+func CreateClusterTemplate(t *testing.T, client *gophercloud.ServiceClient) (*clustertemplates.ClusterTemplate, error) {
+	choices, err := clients.AcceptanceTestChoicesFromEnv()
+	if err != nil {
+		return nil, err
+	}
+
+	name := tools.RandomString("TESTACC-", 8)
+	t.Logf("Attempting to create cluster template: %s", name)
+
+	boolFalse := false
+	boolTrue := true
+	createOpts := clustertemplates.CreateOpts{
+		Name:                name,
+		Labels:              map[string]string{},
+		FixedSubnet:         "",
+		MasterFlavorID:      "",
+		NoProxy:             "10.0.0.0/8,172.0.0.0/8,192.0.0.0/8,localhost",
+		HTTPSProxy:          "http://10.164.177.169:8080",
+		TLSDisabled:         &boolFalse,
+		KeyPairID:           "kp",
+		Public:              &boolFalse,
+		HTTPProxy:           "http://10.164.177.169:8080",
+		DockerVolumeSize:    3,
+		ServerType:          "vm",
+		ExternalNetworkID:   choices.ExternalNetworkID,
+		ImageID:             choices.ImageID,
```

Comment:
Usually the `ImageID` is set to a "cirros" image. I don't think the standard Magnum templates will allow this -- they're looking for a Fedora Atomic image.

I think we should add a new Acceptance Test variable of `MagnumImageID` (`OS_MAGNUM_IMAGE_ID`) here: 

https://github.com/gophercloud/gophercloud/blob/master/acceptance/clients/clients.go#L18
https://github.com/gophercloud/gophercloud/blob/master/acceptance/clients/clients.go#L50
https://github.com/gophercloud/gophercloud/blob/master/acceptance/clients/clients.go#L100

But don't return an error if it's missing -- we'll have to update existing test infrastructure to account for this before we can make it a required setting.

Reply:
Will add the new env var OS_MAGNUM_IMAGE_ID